node_modules/col-categories/lib/api.js

```diff
@@ -148,22 +158,28 @@ var getCategoriesById = module.exports.getCategoriesById = function(ctx, ids, ca
  *
  * @param  {Context}        ctx                         Standard context containing the current user and the current course
  * @param  {String}         title                       The name of the category
+ * @param  {Boolean}        [visible]                   Whether assets associated to this category will be visible in the Asset Library. By default, this will be set to `true`
  * @param  {Number}         [canvasAssignmentId]        The id of the assignment to which this category is linked
  * @param  {String}         [canvasAssignmentName]      The name of the assignment in Canvas to which this category is linked
  * @param  {Function}       callback                    Standard callback function
  * @param  {Object}         callback.err                An error that occurred, if any
  * @param  {Category}       callback.category           The created category
  */
-var createCategory = module.exports.createCategory = function(ctx, title, canvasAssignmentId, canvasAssignmentName, callback) {
+var createCategory = module.exports.createCategory = function(ctx, title, visible, canvasAssignmentId, canvasAssignmentName, callback) {
+  // Default the visible setting
+  visible = (visible === false) ? false : true;
```

Comment:
Well, let's leave it as is. The pattern is used elsewhere and consistency is more important. Thx